docs/integrations/react-native.md

```diff
@@ -0,0 +1,185 @@
+---
+title: React Native
+head:
+- - meta
+- name: title
+  content: Solana Cookbook | Using React Native with @solana/web3.js
+- - meta
+- name: og:title
+  content: Solana Cookbook | Using React Native with @solana/web23.js
+- - meta
+- name: description
+  content: React Native is an open-source UI software framework used to develop mobile, web and desktop applications by enabling developers to use the React framework along with native platform capabilities
+- - meta
+- name: og:description
+  content: React Native is an open-source UI software framework used to develop mobile, web and desktop applications by enabling developers to use the React framework along with native platform capabilities
+- - meta
+- name: og:image
+  content: https://solanacookbook.com/cookbook-sharing-card.png
+- - meta
+- name: og:image:alt
+  content: Solana splash card
+- - meta
+- name: twitter:card
+  content: summary
+- - meta
+- name: twitter:site
+  content: "@solanacookbook"
+- - meta
+- name: twitter:image
+  content: "https://solanacookbook.com/cookbook-sharing-card.png"
+- - meta
+- name: robots
+  content: index,follow,noodp
+- - meta
+- name: googlebot
+  content: index,follow
+---
+
+# React Native
+
+React Native is an open-source UI software framework used to develop mobile, web and desktop applications by enabling developers to use the React framework along with native platform capabilities.
+
+## How to use @solana/web3.js in a React Native app
+
+In this tutorial you will learn how to create a new React Native app and install and configure the `@solana/web3.js` SDK, and its dependencies. 
+
+If you want to add the `@solana/web3.js` SDK to an existing app, you can skip the first step and start by installing the dependencies.
+
+### Create a new app
+
+We start a new React Native application that uses TypeScript, then `cd` into the project directory, where we will execute the rest of the commands.
+
+```shell
+npx react-native init SolanaReactNative --template react-native-template-typescript
+cd SolanaReactNative
+```
+
+### Install dependencies
+
+Next, we install the dependencies. We install the Solana SDK, and in addition we install a package to patch the `metro` configuration, and two polyfills that patch the React Native environment. 
+
+```shell
+yarn add @solana/web3.js metro-config react-native-get-random-values react-native-url-polyfill
+```
+
+### Update `index.js`
```

Comment:
Any explanation as to why polyfills and metro need to be updated? Would be a good fyi walking through this

Reply:
The `Common issues` section at the end of the doc explains the 'why' in more detail, is that what you're aiming at?

Reply:
Good enough for now. Thanks!